pom.xml

```diff
@@ -25,7 +25,7 @@ under the License.
   <parent>
     <groupId>org.apache.maven.plugins</groupId>
     <artifactId>maven-plugins</artifactId>
-    <version>33</version>
+    <version>34</version>
```

Comment:
upgrading parent is another story, and would be useful to add the project.build.outputTimestamp property

Reply:
done